### DISCUSSION TITLE
[CLOSED] 🆕 "Request" category in discussions?

### DISCUSSION THREAD

**Discussion by oood** [abbrcode/abbreviations-in-code#85] [Ideas]
Feels like there will be a lot of requests listed here in the future, not sure if requests can be classified as "General discussion".

~~So the Requests category was added.~~

~~https://github.com/orgs/abbrcodes/discussions/categories/requests~~

Deleted

Please share your thoughts on this category. I mean, do you think we need it? Or maybe we need to change the emoji? any thoughts about this category.

**Comment by oood** [comment]
Maybe we don't need "Requests", the ideas is enough? 🤔


Well, I deleted it and reopened this topic if the ideas doesn't satisfy the need.

**Reply by T1xx1** [reply]
The ideas category is enough.

**Reply by oood** [reply]
Ok
